### ISSUE TITLE
Option to define the position of "php-docblocker.extra"

### ISSUE THREAD

**Issue by franzarmas** [neild3r/vscode-php-docblocker#89]
Hello, it would be good if there is an option that defines in which part of the comment will be configured in "php-docblocker.extra".

In my configuration I have this:

```php
"php-docblocker.extra": [
     "@since Unknown",
],
````
And the output of default is:

```php
/**
 * Undocumented function
 *
 * @param [type] $admin_bar
 * @return void
 * @since Unknown
 */
public function admin_bar($admin_bar) {
   # code ...
}
````

But I would like to get this output, that is to say that what is configured in "php-docblocker.extra" comes on top of @param :
```php
/**
 * Undocumented function
 *
 * @since Unknown
 *
 * @param [type] $admin_bar
 * @return void
 */
public function admin_bar($admin_bar) {
   # code ...
}
````

**Comment by neild3r** [comment]
@franzarmas see #98 this PR will hopefully give you the flexibility you want plus more

**Comment by neild3r** [comment]
@franzarmas this is now available in v2.0.0
